src/styles/bloks/product-card.scss

```diff
@@ -0,0 +1,104 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  box-sizing: border-box;
+  width: $product-card-width;
+  border: 1px solid $border-color;
+  border-radius: $border-radius;
```

Comment:
Ensure that the variables `$product-card-width`, `$border-color`, and `$border-radius` are defined in your SCSS files. Without these definitions, the SCSS will not compile correctly.